image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
+Allows for mapping random access to the references in the image-layout, and needing only to read from the `oci-layout` once to build this map of references.
+
+**index** *array of objects*, OPTIONAL
```

Comment:
The ref list could get large, and I'd rather not overload the “do I understand how to parse this?“ `oci-layout`.  Can we put this index in `refs.json` instead?  You could just dive in with the array:

```
$ cat refs.json
["latest", "v1"]
```

Since the filename + the version from `oci-layout` sufficiently namespace it.